google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/JobTest.java

```diff
@@ -246,18 +249,55 @@ public void testWaitForAndGetQueryResults() throws InterruptedException {
     initializeExpectedJob(2, jobInfo);
     JobStatus status = createStrictMock(JobStatus.class);
     expect(bigquery.getOptions()).andReturn(mockOptions);
-    expect(mockOptions.getClock()).andReturn(CurrentMillisClock.getDefaultClock());
+    expect(mockOptions.getClock()).andReturn(CurrentMillisClock.getDefaultClock()).times(2);
     Job completedJob = expectedJob.toBuilder().setStatus(status).build();
-    QueryResponse completedQuery = QueryResponse.newBuilder().setJobCompleted(true).build();
+    // TODO(pongad): remove when https://github.com/googleapis/gax-java/pull/431/ lands.
```

Comment:
Looks like this maybe landed already?

Reply:
This requires bumping dependency version. Let's do this after merge?